project-api/project-api-test/src/test/resources/myproject_with_lang_import/modules/services/svc.bal

```diff
@@ -0,0 +1,5 @@
+import sameera/myproject.storage;
+
+public function runServices() {
+    storage:initDatabase();
+}
```

Comment:
```suggestion
}

```

Reply:
Fixed in 95dc03b438c8c66c9f0ccf3fa841e548ff1688a9 commit